### PR TITLE
chore: create changelog for 41328

### PR DESCRIPTION
**Pull request by timeswind**
This pull request only adds changelog to unrelease folder for recent merged pull request #41328 .

### CODE REVIEW

File: changelog/unreleased/41328 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Enhancement: Improve global search experience for Chinese and Japanese input
+
+Now the minimum charactors input requried is 2 for Chinese/Japanese inputs when perform global search.
```
Comment (phil-davis) [inline changelog/unreleased/41328:3]:
```suggestion
Now the minimum characters input required is 2 for Chinese/Japanese inputs when performing global search.
```

**Comment by phil-davis** [comment]
@timeswind commit messages need to be in the "conventional commits" format.

https://drone.owncloud.com/owncloud/core/40148/3/2

In this case, the commit message could just start with "chore:" - can you change the commit message to something like "chore: create changelog for 41328"

**Comment by phil-davis** [comment]
The commit messages need to be changed, not just the title of the PR.
Do you know how to do that with `git`?
If not, then I can try to do it.

**Comment by timeswind** [comment]
Close this PR for new PR request from new branch #41336

**Comment by timeswind** [comment]
> The commit messages need to be changed, not just the title of the PR. Do you know how to do that with `git`? If not, then I can try to do it. (It would be much easier if your PR was not from the master branch of your fork. Usually you would create a new branch from `master` and make the changes in that branch, and then push and make a PR)

Thanks for your advice, I think I eventually managed to do this.

**Pull request closed without merging**